install/kubernetes/tetragon/templates/operator_clusterrole.yaml

```diff
@@ -26,12 +26,14 @@ rules:
       - patch
       - update
       - watch
+  {{- if not .Values.tetragonOperator.skipCRDCreation }}
```

Comment:
```suggestion
  {{- if eq .Values.crds.installMethod "operator" }}
```
It needs to be changed as `tetragonOperator.skipCRDCreation` value got deprecated & removed in the meantime.

Reply:
Okay I've done the changes as per that.

Reply:
Hey @itsCheithanya I don't see new changes, did you forget to push maybe?

Reply:
looks good now, thanks!